packages/tooling/cfp-template/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@microsoft/cfp-template",
+  "description": "The default FAST project template",
```

Comment:
If this is the first time reading the description, with no previous context, this description isn't clear.  What is the `default`?  Could we say something more as an action?  "Create FAST Projects using our default template for enterprise web solutions" for example.

Reply:
I don't think we should use the word "enterprise".

Reply:
that was just an example.  With this default, what is it?  What does it set me up for?  There could be differences when building for enterprise scale versus a simple marketing site?  Or would that default still apply to both?

Reply:
I am being vague on purpose, because what this template does is similar to create fast app, it just creates a baseline app with predetermined setup. In the future however, we may present various options (see the linked issue). This is also why we are starting at `0.1.0`, which according to semver when you are in a version before `1.0.0` you can make fairly radical changes.